### DISCUSSION TITLE
OpenFreeMap example

### DISCUSSION THREAD

**Discussion by hyperknot** [openlayers/openlayers#16223] [General]
Hi,

I've just launched [OpenFreeMap](https://openfreemap.org/) yesterday. I'd like to include an official OpenLayers example. What would you recommend me for such an example?

I've found the OpenMapTiles example on this page: https://openlayers.org/ol-mapbox-style/examples/

Is that the right way of using it?

**Comment by ahocevar** [comment]
Thanks @hyperknot for launching that service!

To compare best with your MapLibre example, you could use
```html
<script src="https://unpkg.com/ol@10.2.0/dist/ol.js"></script>
<script src="https://unpkg.com/ol-mapbox-style@12.3.5/dist/olms.js"></script>
<link rel="stylesheet" href="https://unpkg.com/ol@10.2.0/ol.css" type="text/css">
```
```html
<div id="map" style="width: 100%; height: 500px"></div>
<script>
  const map = new ol.Map({
    target: 'map',
    view: new ol.View({
      center: ol.proj.fromLonLat([13.388, 52.517]),
      zoom: 9.5
    })
  });
  olms.apply(map, 'https://tiles.openfreemap.org/styles/liberty');
</script>
```
If you want to show the module way, then https://openlayers.org/ol-mapbox-style/examples/tilejson-vectortile.html would be the minimal example to match:
```js
import 'ol/ol.css';
import {apply} from 'ol-mapbox-style';

apply('map', ' https://tiles.openfreemap.org/styles/liberty');
```

**Reply by hyperknot** [reply]
Thanks a lot, I'll include it in the examples!
